src/components/atoms/Button/index.tsx

```diff
@@ -1,9 +1,14 @@
-import React, {ReactNode} from 'react'
+import { ReactNode } from 'react'
 import './style.scss'
```

Comment:
Почему не используешь css-modules?

```suggestion
import styles from './style.scss'
```